charts/icm-as/values.yaml

```diff
@@ -21,18 +21,13 @@ podBinding:
 
 # define custom labels for deployment and pods:
 deploymentLabels: {}
-#  tags.datadoghq.com/env: hell
-#  tags.datadoghq.com/service: icm-as
-#  tags.datadoghq.com/version: 11.0.0
 podLabels: {}
-#  tags.datadoghq.com/env: hell
-#  tags.datadoghq.com/service: icm-as
-#  tags.datadoghq.com/version: 11.0.0
 
 image:
   pullPolicy: IfNotPresent
   # 'appServer' container provides the application / service endpoint
-  repository: "intershophub/icm-as:11.0.8"
+  repository: "intershophub/icm-as"
```

Comment:
Requires to also adapt all umbrella-charts e.g. the `icm`-chart (but can also be separately released)